testdata/file2.txt

```diff
@@ -0,0 +1 @@
+foo2
```

Comment:
Are the contents of these files important at all? Should/could we just write tmp files to disk in the test setup and then remove them in the teardown?

Reply:
I think this is simpler than setting up the test directory structure before every test. It's especially important for testing walkDirectory